src/clj/commiteth/eth/token_registry.clj

```diff
@@ -1,6 +1,7 @@
 (ns commiteth.eth.token-registry
   (:require [commiteth.eth.core :as eth]
             [commiteth.config :refer [env]]
+            [commiteth.eth.web3j :refer [web3j-obj creds-obj]]
```

Comment:
Not important but I generally avoid `:refer` as it makes it harder to see where stuff is coming from.